profile/README.md

```diff
@@ -0,0 +1,28 @@
+## Welcome to the Puppet community 👋
+
+![Puppet folk having a grand time at Portland's annual Pride parade.](https://raw.githubusercontent.com/puppetlabs/.github/org-profile/profile/Community-Marquee-1792x768.png)
+
+At Puppet, open source software is in our DNA. From the earliest days of Facter to the latest version of Bolt, we’ve always been firm believers in the power of open source. We have a supportive, active community of thousands of people just like you who are making Puppet better — and making IT a better place to work.
+
+
+### 🎪 An interconnected community
+
+You can get involved in the [Puppet community](https://puppet.com/community/) by connecting at events, getting or giving help in any of our many channels, sharing modules on the Forge, contributing code, docs, or any of your valuable insights--and many other ways, too! However you decide to jump in, we know you’ll learn a lot and enjoy meeting the smart, friendly people who make up the Puppet community. Maybe you'll even fall in with our good friends at [🦊 Vox Pupuli](https://voxpupuli.org)!
+
+- 💬 https://slack.puppet.com
+- 🎫 https://tickets.puppet.com
+- 🎩 https://forge.puppet.com
+- 🦊 https://voxpupuli.org
+- 🎙 https://dev.to/puppet
+
+Don't forget that all participation is governed by our [Code of conduct](https://pup.pt/conduct).
+
+### 🎁 Contributing to the ecosystem
+
+Community contributions are awesome and part of the way that we make sure that our tools continue to solve your problems. We want to keep it as easy as possible to participate and make contributions. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things, of course!
+
+Most of our projects will use our standard [contribution guidelines](https://github.com/puppetlabs/.github/blob/master/CONTRIBUTING.md). but you should check individual repositories for a more specific guide before submitting a PR.
```

Comment:
ooh should we make the `master`->`main` switch for this repo?

Reply:
good call! I'll do that separately